Cloud Application Manager/Core Concepts/providers.md

```diff
@@ -45,8 +55,45 @@ Every provider in the list have a gear icon from which you can synchronize or de
 
 If you select several providers by clicking on the checkbox on the provider line item (or clicking the checkbox in the header line which will select all), the **Bulk Actions** button dropdown will appear next to the *New* provider button, allowing to sync or delete the selected providers in bulk mode.
 
+
+### Synchronization process
+
+
+Cloud Application Manager requires to gather specific information of your cloud providers to be able to provision and orchestrate deployments automatically. The basic synchronization of a provider created from an existing account is a read only process which do not change anything in your cloud account.
+
+After you register a provider, Cloud Application Manager uses API calls to retrieve metadata which describes available infrastructure options, quotas, your custom configuration and images or templates to store it with the purpose of provide such customized options to you when you are creating or editing a deployment policy box.
+
+This metadata includes required infrastructure data as follows:
+* Available regions or datacenters
+* Available networks and subnetworks
+* Allowed connectivity options with public or private ip and firewall rules
+* Available autoscalability configuration (i.e. resource limits or alarms)
+* Allowed type/size of virtual machines
+* Available images or templates to be used at provision time. These are listed in configuration of your provider
+* Available public keys to secure provisioned virtual machines
+* Available load balancers and target groups
+* Unregistered virtual machines and autoscaling groups which are already provisioned in your cloud account in order to allow you to register them if you want to do so. These are listed in Unregistered tab of your provider. Discovery of these unregistered virtual machines can be done atomatically if you selected so in your provider Services tab: [Automatic Discovery of Resources](https://www.ctl.io/knowledge-base/cloud-application-manager/getting-started/register-existing-instance/#discovering-the-unregistered-instances)
```

Comment:
I would also add that other native resources are discovered and displayed in the Instances page with some basic information.

Reply:
done